client/src/utils/plot.js

```diff
@@ -152,12 +156,13 @@ export default (elementId, plotData) => {
   const datasets = Object.values(plotData.plot_data);
 
   // Build the color generator for the lines and legend
-  const colors = datasets.map((val, i, arr) => d3.rgb(
+  const colors = datasets.map((val, i) => d3.rgb(
     // The different color options are here: https://github.com/d3/d3-scale-chromatic
-    d3.interpolateCool(i / arr.length),
+    d3.interpolateCool(i / datasets.length),
   ).formatHex());
+  debug('colors are: ', colors);
```

Comment:
Idk much about this debug function.. should we get rid of this before we merge?

Reply:
They don't take effect unless `debug.enabled = true` at the top of the file. [Link to the package](https://www.npmjs.com/package/debug).